scripts/update_package_cache.py

```diff
@@ -63,7 +63,7 @@ def update_test_packages_cache(package_list_dir_path: Path, pipx_package_cache_p
 
     if not platform_package_list_path.exists():
         print(
-            f"WARNING.  File {str(platform_package_list_path)}\n" "    does not exist.  Creating now...",
+            f"WARNING.  File {str(platform_package_list_path)}\n    does not exist.  Creating now...",
```

Comment:
Not sure what to do here. In #1118 / 7184de2 the `black` run folded this multiline string:
```python
        print(
            f"WARNING.  File {str(platform_package_list_path)}\n"
            "    does not exist.  Creating now...",
            file=sys.stderr,
        )
```
into:
```python
        print(
            f"WARNING. File {str(platform_package_list_path)}\n" " does not exist. Creating now...",
            file=sys.stderr,
        )
``` 

Reverting is not an option, as the next run of `black` will overwrite once again.

Perhaps use multiline strings?
```python
        print(
            f"""\
WARNING.  File {str(platform_package_list_path)}"
    does not exist.  Creating now..."'',
            file=sys.stderr,
        )
```